xbmc/video/ContextMenus.cpp

```diff
@@ -119,6 +122,39 @@ bool CVideoMarkUnWatched::Execute(const std::shared_ptr<CFileItem>& item) const
   return true;
 }
 
+bool CVideoBrowse::IsVisible(const CFileItem& item) const
+{
+  if (item.IsFileFolder(EFILEFOLDER_MASK_ONBROWSE))
+    return false; // handled by CMediaWindow
+
+  return item.m_bIsFolder && VIDEO_UTILS::IsItemPlayable(item);
+}
+
+bool CVideoBrowse::Execute(const std::shared_ptr<CFileItem>& item) const
+{
+  int target;
```

Comment:
Does it make sense to initialize this to some invalid value instead of keeping the default 0 value?

Reply:
'target' safely gets assigned a value if .. else if .. else, all branches assign `target`a value, but I can change that, for safety, yes.